.circleci/config.yml

```diff
@@ -86,6 +86,17 @@ commands:
           path: result-reports
       - store_test_results:
           path: result-reports
+  skip_on_push_builds:
+    steps:
+      - unless:
+          condition:
+            matches:
+              pattern: ^pull/\d+$
```

Comment:
This pattern won't work in all cases because PR branches from forks start with `pull/`, while PR branches on the main repo can have arbitrary names like `danielrozenberg-patch-1`. I've suggested an alternative below.

Reply:
Done.